variables.tf

```diff
@@ -34,6 +34,11 @@ variable "jenkins_AAD_objectId" {
 }
 
 // ASP Specific Variables
+variable "asp_sku_size" {
+  type    = "string"
+  default = "I3"
```

Comment:
default  I3? is this only for AAT? if is the case is possible to remove this config from each project?

Reply:
Unfortunately not, this ASP code only takes effect if this build is run. Otherwise the individual repos are going to keep resizing everytime they are built. But good catch on default, I'll default it to I2 preferrably and set in the individual tfvars the proper size.